src/main/eventHandlers/willDownloadEvents/setInterruptedDownload.ts

```diff
@@ -0,0 +1,28 @@
+// @ts-nocheck
+
+import downloadStates from '../../../app/constants/downloadStates'
+
+/**
+ * If no more files are active, mark the download as interrupted
+ * @param {Object} params
+ * @param {Object} params.database `EddDatabase` instance
+ * @param {Object} params.downloadId downloadId of the DownloadItem being downloaded
+ */
+const setInterruptedDownload = async ({
+  database,
+  downloadId
+}) => {
+  const activeFilesCount = await database
+    .getActiveFilesCountByDownloadId(downloadId)
+
+  // If no files remain that aren't interrupted or
```

Comment:
```suggestion
  // If no active files remain
``` ?
The `getActiveFilesCountByDownloadId` just checks if any files are in 'active' state